CONTRIBUTING.md

```diff
@@ -235,6 +235,9 @@ wiki](https://github.com/oamg/convert2rhel/wiki).  When you add a new feature th
 to plug into or affects how their code should be written in the future, please add a new page there
 and link it to the toplevel [Topics section](https://github.com/oamg/convert2rhel/wiki#topics).
 
+### Type comments
```

Comment:
Should we also include in this text about types in docstrings?

Reply:
Yeah, or if we have docstrings mentioned somewhere we can move this to be under that